### PR TITLE
create vacation manager

### PR DESCRIPTION
**Pull request by thiennguyen96841996**

### CODE REVIEW

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Controllers/Manager/ManagerVacationController.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $date = Carbon::now();
+        $works = Working::where('status', 1)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->whereDay('date', Carbon::now()->format('d'))->get();
+        $lates = Working::where('status', 0)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->whereDay('date', Carbon::now()->format('d'))->get();
+        $statics = Working::where('status', 1)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->get()->unique('user_id');
+        return view('backend.users.vacations.index', compact('date', 'works', 'lates', 'statics'));
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/ManagerVacationController.php]: thiếu dòng trắng trước return

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

File: app/Http/Controllers/Manager/ManagerVacationController.php [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $date = Carbon::now();
+        $works = Working::whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->whereDay('date', Carbon::now()->format('d'))->get();
+        $lates = Working::where('status', 0)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->whereDay('date', Carbon::now()->format('d'))->get();
+        $statics = Working::where('status', 1)->whereMonth('date', Carbon::now()->format('m'))->whereYear('date', Carbon::now()->format('Y'))->get()->unique('user_id');
+        $vacations = Vacation::where('status', 1)->where('date_start', 'like', '%'.Carbon::now()->toDatestring('Y:m:d').'%')->where('date_end', 'like', '%'.Carbon::now()->toDatestring('Y:m:d').'%')->get()->unique('user_id');
```
Comment (BunnyPi04) [inline app/Http/Controllers/Manager/ManagerVacationController.php]: convention

**Comment by BunnyPi04** [comment]
commented

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Comment by thiennguyen96841996** [comment]
ready

**Pull request merged**